runtests

```diff
@@ -27,13 +27,11 @@ main() {
   done
 
   echo 'Running shellcheck'
-  git grep -l '^#!/usr/bin/env bash' | xargs shellcheck
-  # Temporarily disable shellcheck checks for SC2154 (undefined variable) on /bin/bash files
-  git grep -l '^#!/bin/bash' | xargs shellcheck -e SC2154
+  # Note: shellcheck checks for SC2154 (undefined variable) are disabled by '-e SC2154' below
+  git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck -e SC2154
```

Comment:
I want to make sure we still check for undefined variables most of the time.  It's not documented here, but this was my original intent with using different shebangs (yes, a not great decision...).  I would prefer all files to manage individual shellcheck rules locally via comments.

Reply:
@meatballhat agreed; I did this as a workaround for the fact that some of the `.sh` files that weren't getting checked before raised `shellcheck` errors that I don't know how to fix.

Reply:
Perhaps instead individual SC2154 occurrences could be ignored using the per-file annotations?
https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-one-specific-instance-in-a-file
https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-all-instances-in-a-file-044

Reply:
Oh nice!